spec/furniture/marketplace/products_controller_request_spec.rb

```diff
@@ -36,18 +36,19 @@
     end
 
     describe "when product is invalid" do
-      let(:product_attributes) { {} }
+      let(:product_attributes) { {description: "test"} }
 
       it { is_expected.to have_rendered(:new) }
     end
   end
 
   describe "#edit" do
     it "Shows the form to edit a Marketplace Product" do
+      sign_in(space, member)
```

Comment:
!!! Apparently we were not authorizing the `edit` action until after we hit the view; which was allowing this test to "pass" even though no one was signed in...

Reply:
Good catch.